Gemfile

```diff
@@ -7,6 +7,8 @@ gem "ohai", git: "https://github.com/chef/ohai.git", branch: "master"
 gem "chef-utils", path: File.expand_path("chef-utils", __dir__) if File.exist?(File.expand_path("chef-utils", __dir__))
 gem "chef-config", path: File.expand_path("chef-config", __dir__) if File.exist?(File.expand_path("chef-config", __dir__))
 
+gem "corefoundation", git: "https://github.com/chef/corefoundation.git"
```

Comment:
I doubt this gem is ever going to get updated by the author.

It would be better to:  1.  break the fork on github (forked repos are not searchable the same way that the source repo is) and 2.  upload to rubygems.org under another name like chef-corefoundation.